adapters/repos/db/shard_write_inverted_lsm.go

```diff
@@ -87,8 +88,18 @@ func (s *Shard) addToPropertyValueIndex(docID uint64, property inverted.Property
 		if bucketValue == nil {
 			return errors.Errorf("no bucket searchable for prop '%s' found", property.Name)
 		}
+		propLen := float32(0)
 
-		propLen := float32(len(property.Items))
+		if os.Getenv("USE_INVERTED_SEARCHABLE") == "true" || os.Getenv("USE_NEW_PROPLENGTH") == "true" {
```

Comment:
I'd use a more descriptive naming for `USE_NEW_PROPLENGTH`

Reply:
Yeah, I'm still debating on how to make this accessible.